spec/factories/users.rb

```diff
@@ -32,8 +32,75 @@
     last_name { Faker::Name.last_name }
     sequence(:email) { |index| Faker::Internet.email.sub(/\@/, "_#{index}@") }
     password { Faker::Internet.password }
-    role { Faker::Job.title }
+    role { "user" }
     department { Faker::Job.field }
     about { Faker::Lorem.sentences }
   end
-end
+
+  trait :admin do
+    role { "admin" }
```

Comment:
I think we have this field somewhere on UI

so we need to remove it from UI forms (in profile)

or rename in profile to "job_role"